.pylintrc

```diff
@@ -1,3 +1,11 @@
+# Copyright (c) 2022, Hewlett Packard Enterprise Development Company LP
```

Comment:
Copyright year needs update

Reply:
Might want to check where you cloned the copyright statement from as well

Reply:
Great catch. Done in both places